securedrop_client/storage.py

```diff
@@ -559,12 +565,11 @@ def update_replies(
 
             del local_replies_by_uuid[reply.uuid]
```

Comment:
This is easier to reason about now since we want `local_replies_by_uuid` to end up with _all_ the local replies that don't exist on the server.